src/adc.rs

```diff
@@ -98,6 +135,126 @@ impl Adc {
         self.precision = precision;
     }
 
+    /// Starts a continuous conversion process
+    ///
+    /// The `channel` argument specifies which channel should be converted.
+    ///
+    /// The `trigger` argument specifies the trigger that will start each
+    /// conversion sequence. This only configures the ADC peripheral to accept
+    /// this trigger. The trigger itself must also be configured using its own
+    /// peripheral API.
+    ///
+    /// In addition to the preceeding arguments that configure the ADC,
+    /// additional arguments are required to configure the DMA transfer that is
+    /// used to read the results from the ADC:
+    /// - `dma` is a handle to the DMA peripheral.
+    /// - `dma_chan` is the DMA channel used for the transfer. It needs to be
+    ///   one of the channels that supports the ADC peripheral.
+    /// - `buffer` is the buffer used to buffer the conversion results.
+    ///
+    /// # Panics
+    ///
+    /// Panics, if `buffer` is larger than 65535.
+    #[cfg(feature = "stm32l0x2")]
+    pub fn start<DmaChan, Buf>(mut self,
+        channels: impl Into<Channels>,
+        trigger:  Option<Trigger>,
+        dma:      &mut dma::Handle,
+        dma_chan: DmaChan,
+        buffer:   Pin<Buf>,
+    )
+        -> Adc<Active<DmaChan, Buf>>
+        where
+            DmaToken:    dma::Target<DmaChan>,
+            Buf:         DerefMut + 'static,
+            Buf::Target: AsMutSlice<Element=u16>,
+            DmaChan:     dma::Channel,
+    {
```

Comment:
Sorry for bikeshedding (I'm not qualified enough to review the actual logic), but do we apply rustfmt to this project?

Reply:
No, not as a general rule.